cross/par2cmdline-turbo/Makefile

```diff
@@ -0,0 +1,41 @@
+PKG_NAME = par2cmdline-turbo
+PKG_VERS = 1.1.0
+PKG_EXT = tar.gz
+PKG_GIT_HASH = e00677a8249f438f0f52b0270af9e8c7116eeac0
+PKG_DIST_NAME = $(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/animetosho/par2cmdline-turbo/archive
+PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
```

Comment:
Why don't you use a regular version? Those are available like 
https://github.com/animetosho/par2cmdline-turbo/archive/refs/tags/v1.1.0.tar.gz

I highly recommend to use official versions only (when available).
If you need a fix (commit) that is not already included in the latest available version (and releases are maintained regularly), then you sould take the latest version and apply the fix with a patch (or wait for a version that includes the fix).

Reply:
The author applied fixes for us to get it building on 6.2 architectures and then also qoric.
https://github.com/animetosho/par2cmdline-turbo/issues/12
https://github.com/animetosho/par2cmdline-turbo/issues/22

Those are basically the only changes he made since the latest release.